imagemounter/volume_system.py

```diff
@@ -225,6 +240,17 @@ def _find_pytsk3_volumes(self, vstype='detect'):
                 baseimage.close()
                 del baseimage
 
+    def _detect_exfat_superfloppy(self):
+        """'Detects' a single exFAT superfloppy volume."""
+        f = open(self.parent.get_raw_path(), 'rb')
+        bytes = f.read(11)
+        f.close()
+        if 'EXFAT' in bytes.decode('latin_1'):
```

Comment:
This will probably fail if it can not be decoded to latin-1... Perhaps `b'EXFAT' in bytes`?